frontend/components/onboarding/AccountPassword.tsx

```diff
@@ -124,17 +121,28 @@ export const AccountPassword = (props: AccountPasswordProps) => {
             }}
           ></div>
         </div>
-        <div className="flex w-full items-center gap-4 p-3 bg-zinc-200 dark:bg-zinc-800 dark:bg-opacity-60 rounded-b-md text-black/50 dark:text-white/50">
-          {passwordIsStrong ? <FaCheck /> : <FaInfo />}
-          {passwordIsStrong ? 'Strong password' : pwStrength?.feedback?.suggestions}
+        <div className="flex w-full items-start gap-4 p-3 bg-zinc-200 dark:bg-zinc-800 dark:bg-opacity-60 rounded-b-md text-black/50 dark:text-white/50">
+          <div className="mt-1">
+            {passwordIsStrong ? <FaCheck /> : <FaInfo />}
+          </div>
+          <div className="flex-grow">
+            {passwordIsStrong ? (
+              'Strong password'
+            ) : (
+              <ul className="list-disc pl-5 space-y-1">
+                {pwStrength?.feedback?.suggestions?.map((suggestion, index) => (
+                  <li key={index}>{suggestion}</li>
+                ))}
+              </ul>
+            )}
+          </div>
```

Comment:
```suggestion
        <div className="flex w-full items-start gap-6 p-3 bg-zinc-200 dark:bg-zinc-800 dark:bg-opacity-60 rounded-b-md text-black/50 dark:text-white/50 text-sm">
          <div className="mt-1">
            {passwordIsStrong ? <FaCheck /> : <FaInfo />}
          </div>
          <div className="flex-grow">
            {passwordIsStrong ? (
              'Strong password'
            ) : (
              <ul className="list-disc pl-4">
                {pwStrength?.feedback?.suggestions?.map((suggestion, index) => (
                  <li key={index}>{suggestion}</li>
                ))}
              </ul>
            )}
          </div>
```

This works, just a few suggestions on the styling:
* Smaller font size
* Reduced spacing between suggestions
* Tweaked horizontal padding on bullet list